contribs/gmf/src/search/component.js

```diff
@@ -420,6 +420,7 @@ gmf.search.component.SearchController_ = class {
     const gettextCatalog = this.gettextCatalog_;
     this.clearButton = this.clearButton !== false;
     this.colorChooser = this.colorChooser === true;
+    this.delay = parseInt(this.delay, 10);
```

Comment:
@gberaudo is this parseInt requied?

Reply:
@sbrunner, maybe the value is sometime `undefined`?
I would expect the following to be more correct:
```
if (this.delay === undefined) {
  this.delay = 50;
}
```